apps/mobile/src/store/settings/utils.ts

```diff
@@ -9,7 +9,7 @@ import {
   FiatCurrency,
 } from '@leather.io/models';
 
-export const defaultNetworkPreferences = ['mainnet', 'testnet', 'signet'] as const;
+export const defaultNetworkPreferences = ['mainnet', 'testnet4', 'signet'] as const;
```

Comment:
Just double checking is `testnet4` correct here?

Reply:
yep